src/visualization/interaction/OrbitControls.js

```diff
@@ -346,9 +346,9 @@ export class OrbitControls extends THREE.EventDispatcher {
 
         event.preventDefault();
       }
-    }
+    };
 
-    function onTouchEnd(event3D) {
+    const onTouchEnd = (event3D) => {
```

Comment:
This fixes a bug: the `function`s were passed unbound as event handlers further below, so `this` would not work inside of them.

We need unit tests! ros3d is hardly covered.